bigtable-hbase-1.0/pom.xml

```diff
@@ -155,13 +156,24 @@ limitations under the License.
                                     <include>com.google.oauth-client:*</include>
                                     <include>com.google.protobuf:*</include>
                                     <include>com.twitter:hpack</include>
-                                    <include>com.fasterxml.jackson.core:*</include>
                                     <include>io.netty:*</include>
                                     <include>io.grpc:*</include>
                                 </includes>
                             </artifactSet>
                             <relocations>
                                 <relocation>
+                                    <pattern>com.fasterxml</pattern>
+                                    <shadedPattern>com.google.bigtable.repackaged.com.faster.xml</shadedPattern>
+                                </relocation>
```

Comment:
com.twitter is added. Does it mean that CBT and HBase are using two different incompatible versions?